src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -210,84 +209,73 @@ public boolean test(final GATKRead read){
     private static final class SVCustomReadFilter implements ReadFilter{
         private static final long serialVersionUID = 1L;
 
-        private final ReadFilter collapsedFilter;
+        private final ReadFilter combinedReadFilter;
 
-        public SVCustomReadFilter(final UseEnd whichEnd,
-                                  final boolean filterNonProperlyPairdReads,
-                                  final boolean filterUnmappedReads,
-                                  final boolean filterMateUnmappedReads,
-                                  final boolean filterDupReads,
+        public SVCustomReadFilter(final EndToUse whichEnd,
+                                  final boolean filterNonProperlyPairedReads,
+                                  final boolean filterDuplicatedReads,
                                   final boolean filterSecondaryAlignments,
                                   final boolean filterSupplementaryAlignments,
-                                  final int MQThreshold){
-
-                final UseEnd endVal = whichEnd;
+                                  final int     MQThreshold,
+                                  final int     mMQThreshold){
 
-                ReadFilter tempFilter = read -> 0!=read.getFragmentLength();
```

Comment:
As mentioned in person, I was just thinking that since this is its own class it should probably be self-sufficient and not depend on other filters having been applied.